client/gutenberg/editor/controller.js

```diff
@@ -62,6 +65,48 @@ function waitForSiteIdAndSelectedEditor( context ) {
 	} );
 }
 
+function handleJetpackSSO( context ) {
+	// If we are dealing with an Atomic or Jetpack site we need to make sure that Jetpack SSO
+	// has been handled before we load the editor in an iFrame in order to prevent any issues
+	// with 3rd party cookie setting being blocked by the browser
+	const state = context.store.getState();
+	const currentRoute = getCurrentRoute( state );
```

Comment:
Inside a page.js handler, it's better to use page.js API instead of reading from Redux. The current route is in `context.pathname`.

Reply:
With the refactor done in 136a57d3a6c74ae86bdbaa9e2f87ba4f8ff24149 we don't the current route anymore.